gwsumm/plot/segments.py

```diff
@@ -1352,7 +1360,7 @@ def draw(self, outputfile=None):
 
         # make bar chart
         width = plotargs.pop('width', .8)
-        x = numpy.arange(len(data)) - width/2.
+        x = numpy.arange(len(data))
```

Comment:
Does this change impact any of the other plots on the summary pages?

Reply:
This is just compensating for the removal of line 1289. Line 1289 was setting the bars to align at the edge, then -width/2 was moving x to the center.

Reply:
Ok, sounds good!